src/libsyntax/feature_gate.rs

```diff
@@ -854,6 +851,8 @@ declare_features! (
     (accepted, repr_align_enum, "1.37.0", Some(57996), None),
     // Allows `const _: TYPE = VALUE`.
     (accepted, underscore_const_names, "1.37.0", Some(54912), None),
+    // Allows `#[doc(alias = "...")]`.
+    (accepted, doc_alias, "1.27.0", Some(50146), None),
```

Comment:
```suggestion
    (accepted, doc_alias, "1.41.0", Some(50146), None),
```

Reply:
Sill waiting for approval before rebasing. It's just one of the conflicts. ;)

Reply:
Hehe; I just did this for all 1.40.0 PRs that were relevant ^^

Reply:
:p